web/src/components/buttontoggle.js

```diff
@@ -40,11 +41,17 @@ const ButtonToggle = ({
         onClick={onChange}
         active={active}
         icon={icon}
+        errorMessage={errorMessage}
       />
       {tooltipVisible && !isEmpty(tooltip) && (
         <div className="layer-button-tooltip">
           <div className="tooltip-container">
-            <div className="tooltip-text">{tooltip}</div>
+            <div className="tooltip-text">
+              { !errorMessage && (<div>{tooltip}</div>)}
+              { active && errorMessage && (
+                <div className='tooltip-error'>{errorMessage}</div>
+              )}
```

Comment:
Right now the error message is only showing when the button is active, so we're missing the "it's unavailable" message. 

![Screenshot 2021-08-11 at 14 17 24](https://user-images.githubusercontent.com/3296643/129027314-0a8a57e2-798b-4445-ab2e-89dcb81161c4.png)

This change will make it appear no matter of it's active or not :)

```suggestion
              { errorMessage && (
                <div className='tooltip-error'>{errorMessage}</div>
              )}
```